docs/node/migrations/zero-downtime-migrations/index.mdx

```diff
@@ -0,0 +1,135 @@
+            "end_height": <end_height>,
+        },
+    }
+}
+```
+:::warning
+`start_height` and `end_height` in the genesis file and `migrate_from` in the config file are mandatory to ensure that the new network stays in sync with the existing network during the migration period. If neither are set, the new network will start with the state from the snapshot state, but doesn't include any state changes from the existing network. If one of them is set but not other, the node will fail to start.
+:::
+
+#### Handling State Divergence
+
+During migration period, as both old and new networks are running concurrently handing user transactions and replicating state changes from the existing network, there is a possibility for the occurance of state divergences between the two networks. 
+The migration process does its best to resolve the conflicts between the two networks using a set of conflict resolution principles. However, there might be some schemas where the conflicts cannot be resolved without any loss of data.
+
+:::note
+** Do's and Dont's: ** Operators should migrate to new network as soon as possible and redirect all the user transactions to the new network. Any single user transactions should not be scattered across both new and old chains to avoid any state divergence and should be redirected to a single network as much as possible.
+:::
+
+User Dataset changeset conflicts are handled as follows:
+
+- **Inserts**: Inserts from the old network will attempt to be applied to the new network, and any conflicts will be ignored, in favor of whatever already exists on the new network.
+- **Updates**: The new network will check the pre-change value of the proposed update from the old network's changeset. If the new network's record for that primary key matches the pre-change value in the changeset, it will apply the change. If the values are not equal, it will discard the change, in favor of what data exists on the new network
+- **Deletes**: Similar to update, if the record matches the pre-change value in the changeset, it will be deleted. Otherwise, no change will be applied.
+
+All spends from the old network should be replicated to the new network and the conflicts are handled as follows:
+- If a spend from the old network would cause a user's balance to drop below zero, the entire balance will be spent, setting it to zero.
+- Otherwise, the specified amount from the old network’s spend will be deducted from the user’s balance on the new network.
+
+## Migration Ends
+
+The existing network halts after the `Duration` blocks since the start of the migration. The existing network will no longer accept new transactions or produce new blocks with transactions. The RPC services remain available for querying the state of the network. Once the new network is fully synced with the existing network, the existing network can be safely decommissioned. Whereas, the new network continues to produce blocks and process transactions as usual.
```

Comment:
Small grammar:


```suggestion
The existing network halts after the `Duration` blocks since the start of the migration. The existing network will no longer accept new transactions or produce new blocks with transactions. The RPC services remain available for querying the state of the network. Once the new network is fully synced with the existing network, the existing network can be safely decommissioned. The new network will continue to produce blocks and process transactions as usual.
```